src/openapi/streaming/connection/transport/websocket-transport.ts

```diff
@@ -220,7 +226,13 @@ class WebsocketTransport implements StreamingTransportInterface {
                     );
                 }
             }
-            this.lastMessageId = messageId;
+
+            if (

```

Comment:
does it look neater if this is a if around this and the section above?

Reply:
yes i have changed as suggested